openedx/features/marketplace/models.py

```diff
@@ -1,6 +1,33 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from django.contrib.auth.models import User
 from django.db import models
 
-# Create your models here.
+from openedx.custom.db.fields import MultiSelectWithOtherField
+from openedx.features.idea.models import VisualAttachment, Location, OrganizationBase
+from openedx.features.marketplace.constants import (ORGANIZATION_SECTOR_CHOICES, ORGANIZATIONAL_PROBLEM_CHOICES,
+                                                    USER_SERVICES)
+
+
+class Challenge(OrganizationBase, Location, VisualAttachment):
+    user = models.ForeignKey(User, related_name='challenges', related_query_name='challenge', on_delete=models.CASCADE)
+    organization_sector = MultiSelectWithOtherField(other_max_length=50, choices=ORGANIZATION_SECTOR_CHOICES,
+                                                    verbose_name='Which sector is your organization working in?')
+
+    organizational_problems = MultiSelectWithOtherField(other_max_length=50, choices=ORGANIZATIONAL_PROBLEM_CHOICES,
+                                                        verbose_name='Current Organizational Problems')
+
+    description = models.TextField(blank=False, verbose_name='Brief Description of Challenges')
+
+    approch_to_address = models.TextField(blank=False,
```

Comment:
I guess typo, `approach`

Reply:
This is not a required field, all required fields has `*` with them in the form. so blank=True